contribute/issues.md

```diff
@@ -1,39 +1,109 @@
+| kind/feature  | New / existing / required feature or enhancement |
+| kind/improve | Needs improvement |
+| kind/e2e-test | Need/Add end to end test |
+| kind/unit-test | Need/Add Unit test|
+| kind/security  | Security related concerns  |
+| kind/testing  |  Needs/Adds tests |
+
+---
+
+### Help
+
+Labels prefixed with *help* implies that issue needs some help from community/users. Further, you can adjust the level of help required, by specifying small, medium, large etc.
+
+| Label | Description |
+|---|---|
+| help/small  | Need small help on docs (beginner level)|
+| help/medium | Need help to fix code ( Intermediate level) |
+| help/large | Need help on design changes and code review (Expert level) |
+
+---
+
+## Area
+
+Labels prefixed with *area* indicates that which area issue or PR belong. Here, area implies component of the OpenEBS project. It will be easier to sort an issue based on the area of user's interest.
+
+| Label | Description |
+|---|---|
+| area/maya  | Maya Specific  |
+| area/k8s | Kubernetes specific   |
+| area/ci  | Continuous integration specific  |
+| area/high-availability  | High availability  |
+| area/storage | Storage specific  |
+| area/ML  |  Machine learning specific |
+
+---
+
+### Size
+
+Labels prefixed with *size* defines the size of change done in particular PR.
+
+| Label | Description |
+|---|---|
+| size/L | Size of change is large |
+| size/M | Size of change is medium |
+| size/S | Size of change is small |
+| size/XL | Size of change is extra large |
+| size/XS | Size of change is extra small |
+| size/XXL | Size of change is very large |
+
+---
+
+### Priority
+
+Labels with prefix *priority* defines the severity of the issue. Severity level of an issue can be given as a suffix.
+
+| Label | Description |
+|---|---|
+| priority/0 | Urgent : Must be fixed in the next build |
+| priority/1 | High  : Must be fixed in any of the upcoming builds but should be included in the release. |
+| priority/2 | Medium: May be fixed after the release / in the next release |
+| priority/3 | Low: May or may not be fixed at all |
+
+---
+
+### Repo
+
+Labels with prefix *repo* points the repository where given issue needs to be fixed. If users have no idea where to file the issue then they can file issue in `openebs/openebs` later OpenEBS contributors/authors/owners will tag issue with appropriate repository using *repo* label.
+
+| Label | Description |
+|---|---|
+| repo/maya | Related to openebs/maya repository |
+| repo/mayaserver | Related to openebs/mayaserver repository |
+| repo/jiva | Related to openebs/jiva repository |
+| repo/k8s-provisioner|  Related to openebs/k8s-provisioner repository|
+
+---
+
+### General
+
+As name suggests, this contains general category labels and they are self explanatory.
+
+| Label | Description |
+|---|---|
+| architecting | If something needs design changes/brainstorming |
+| documentation  |  Related to documentation |
+| wip:WorkInProgress | Means work in progress |
```

Comment:
do we remove :WorkInProgress (leave it as wip) or call it as work-in-progress

Reply:
`work-in-progress` sounds good